_pages/aleelay.yaml

```diff
@@ -0,0 +1,5 @@
+ ---
+ githubHandle: aleelay
+ pageUrl: https://aleelay.github.io/patchwork/
```

Comment:
Hi @aleelay! Make sure the URL starts with `aleelay`; remove the `https://` so the tests will pass. If this pull request hasn't seen any updates in a few days, one of the GitHub trainers will come around and clean it up to merge it in so we can keep the repository up to date. In the meantime, let me know if you want or need any help. 😄